### DISCUSSION TITLE
GPX infos tags list?

### DISCUSSION THREAD

**Discussion by ghost** [gpx-animator/gpx-animator#554] [Q&A]
The by default tags used as displayed infos are as follow:

```
%SPEED%
%LATLON%
%DATETIME%
```

Can we have a list of all the other tags GPX Animator can use?
I’m fighting to display the Elevation datas and I’m not winnig… :)

With regards.

**Comment by albansuser** [comment]
I agree, that would be very useful. 

For me, I'd like to have "%Distance%" included.

**Comment by joachim1957** [comment]
Yes that would be really great. I'd like to have Elevation and Distance too

**Comment by Hallavera** [comment]
Hello guys. Love the program!!! Please include elevation too is very useful for hiking. Greetings
